pywinauto/controls/HwndWrapper.py

```diff
@@ -463,31 +463,31 @@ def send_message(self, message, wparam = 0, lparam = 0):
     SendMessage = send_message
 
     # -----------------------------------------------------------
-    def send_chars(self, message):
-        """Send a string to the control and wait for it to return"""
-        res = []
+    def send_chars(self,
+                   message,
+                   with_spaces=True,
+                   with_tabs=True,
+                   with_newlines=True):
+        """
+        Silently send a string to the control
 
-        mspl = message.split("{TAB}{ENTER 2}")
-        ind = 0
-        for substr in mspl:
-            ind = ind + 1
+        Parses modifiers Shift(+), Control(^), Menu(%) and Sequences like "{TAB}", "{Enter}"
```

Comment:
I'd prefer describing it as `Alt(%)`. Maybe adding alias `VK_ALT = VK_MENU` to `SendKeysCtypes.py` could be less confusing.